src/lib/tls/msg_client_kex.cpp

```diff
@@ -92,13 +92,6 @@ Client_Key_Exchange::Client_Key_Exchange(Handshake_IO& io,
          if(reader.remaining_bytes())
             throw Decoding_Error("Bad params size for DH key exchange");
 
-         if(p.bits() < policy.minimum_dh_group_size())
```

Comment:
Ok, I think I remove the check from the ServerKeyExchange message and leave the check here.